components/file_system/src/io_stats/biosnoop.rs

```diff
@@ -244,7 +249,8 @@ macro_rules! flush_io_latency {
     };
 }
 
-pub(crate) fn flush_io_latency_metrics() {
+#[allow(dead_code)]
+pub fn flush_io_latency_metrics() {
```

Comment:
I saw it's not used in `components/file_system/src/metrics_manager.rs` why not removed here

Reply:
All codes related to biosnoop is deprecated and useless now. But a lot of efforts went into making it work, so I prefer keeping it for now.

Reply:
Do you mean only `proc::fetch_io_bytes` work now and `biosnoop.rs` deprecated?

Reply:
Yes.